UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -755,11 +756,60 @@ public void Extinguish()
 			healthStateController.SetFireStacks(0);
 		}
 
-		public virtual string GetExamineText()
+		#region Examine
+
+		public string GetExamineText()
 		{
-			return "Weeee";
+			if (this is PlayerHealthV2)
+			{
+				// Let ExaminablePlayer take care of this.
+				return default;
+			}
+
+			// Assume animal
+			var healthString = new StringBuilder("It is ");
+
+			if (IsDead)
+			{
+				healthString.Append("limp and unresponsive; there are no signs of life");
+
+				healthString.Append("...");
+
+				return healthString.ToString();
+			}
+
+			healthString.Append($"{ConsciousState.ToString().ToLower().Replace("_", " ")} and ");
+
+			var healthFraction = OverallHealth / maxHealth;
+			if (healthFraction < 0.2f)
+			{
+				healthString.Append("heavily wounded.");
+			}
+			else if (healthFraction < 0.6f)
+			{
+				healthString.Append("wounded.");
+			}
+			else
+			{
+				healthString.Append("in good shape.");
+			}
+
+			if (respiratorySystem != null && respiratorySystem.IsSuffocating)
+			{
+				healthString.Append("having trouble breathing!");
```

Comment:
this will appear next to the previous line, right after the period.
```suggestion
				healthString.Append(" Having trouble breathing!");
```

Reply:
theres already a space before it in: 

```healthString.Append($"{ConsciousState.ToString().ToLower().Replace("_", " ")} and ");```